package.json

```diff
@@ -37,8 +38,15 @@
   },
   "devDependencies": {
     "@babel/core": "^7.14.3",
+    "@commitlint/cli": "^13.2.0",
+    "@commitlint/config-conventional": "^13.2.0",
+    "@semantic-release/changelog": "^6.0.0",
+    "@semantic-release/exec": "^6.0.1",
+    "@semantic-release/git": "^10.0.0",
     "@types/jest": "^26.0.15",
     "@types/puppeteer": "^5.4.0",
+    "cz-conventional-changelog": "3.3.0",
+    "husky": "^7.0.0",
```

Comment:
In UI Blocks (and other repos), we're using a zero-config solution by just using the `git-commit-msg-linter` package. No need to add commit lint and husky etc.

Reply:
Cool! Replaced husky + commitlint with git-commit-msg-linter 👍